ansible/mnesia_reset_once.yml

```diff
@@ -6,8 +6,30 @@
   vars:
     project_root: "{{ ansible_env.HOME }}/node"
     aeternity_bin: "{{ project_root }}/bin/aeternity"
+    missing_synced_node: true
```

Comment:
this is kind of misleading imho. probably should be reversed like "have_synced_node: false" and later reverse the bool values

Reply:
81d4ea7